config/backstage-plugins.yaml

```diff
@@ -1,6 +1,9 @@
 config:
   branches:
     release-v1.12:
+      konflux:
```

Comment:
I think it might have been better to define ``json:"enabled,omitempty" yaml:"nudges,omitempty"`` here? https://github.com/openshift-knative/hack/pull/161/files#diff-9cb0409bd8a2445656e16ee55c0cc76205f9e9753d4cafa7d59d9bd879395ffdR78

Reply:
@pierDipi

Reply:
Yes, let me fix that

Reply:
https://github.com/openshift-knative/hack/pull/164